versions/1/data_stream/manifest.spec.yml

```
@@ -87,6 +87,14 @@ spec:
       - beta
       examples:
       - beta
+    ilm_policy:
+      description: The name of an existing ILM (Index Lifecycle Management) policy
+      type: string
+      examples:
+      - diagnotics
+    hidden:
+      description: Specifies if a datastream is hidden
```

hidden where?

---

in elastic search

---

I think what @mtojek is suggesting is to include this clarification as part of the `description`.
```suggestion
      description: Specifies if a datastream is hidden in Elasticsearch
```

---

Yes, we definitely need more context here. Could you please justify why does the ILM policy need to be hidden?

---

> hide the data stream from views

Is it hidden in Elasticsearch or Kibana? I'm afraid I'm a bit lost here.

---

@mtojek this field will correspond with this change in ES to give the option to make a `data stream` hidden.  https://github.com/elastic/elasticsearch/pull/63987

Our particular use case will be to define a `data stream` that will collect documents for the sole purpose of telemetry.  As a result, we want the `data stream` to be hidden to reduce user confusion.